### PR TITLE
Update staking-support.md

### PR DESCRIPTION
**Pull request by bjoernek**
Update description of staking maturity in rosetta api

### CODE REVIEW

File: docs/developer-docs/integrations/rosetta/staking-support.md [hunk 1]
```diff
@@ -469,15 +469,17 @@ The `SPAWN` operation creates a new neuron from an existing neuron with enough m
 
 :::
 
-### Merge neuron maturity
+### Stake neuron maturity
 
 |                      |            |
 |----------------------|------------|
 | Since version        | 1.4.0      |
```
Comment (przydatek) [inline docs/developer-docs/integrations/rosetta/staking-support.md:476]: I guess the version may need to be updated.

File: docs/developer-docs/integrations/rosetta/staking-support.md [hunk 2]
```diff
@@ -508,7 +510,7 @@ The `MERGE_MATURITY` operation merges the existing maturity of the neuron into i
 ``` json
 {
   "operation_identifier": { "index": 0 },
-  "type": "MERGE_MATURITY",
+  "type": "STAKE_MATURITY",
   "account": { "address": "907ff6c714a545110b42982b72aa39c5b7742d610e234a9d40bf8cf624e7a70d" },
   "metadata": {
     "neuron_index": 0,
```
Comment (przydatek) [inline docs/developer-docs/integrations/rosetta/staking-support.md:516]: in line 517 we should probably have `percentage_to_stake`.

**Review by przydatek — COMMENTED** [review]
I've added two suggestions, but either way the API owners should approve.

**Comment by jessiemongeon1** [comment]
Applying these changes in new PR https://github.com/dfinity/portal/pull/2635

**Pull request closed without merging**